svelte/components/Global/mediasDisplay/MediasDisplayer.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import { NftType } from "lib/ktypes";
+
+  import { nftGetImageLink } from "lib/knft-get-metadata";
+
+  import MediaPreview from "./mediasComponents/MediaPreview.svelte";
+  import DisplayTypedMedia from "./mediasComponents/DisplayTypedMedia.svelte";
+
+  /////////////////////////////////////////////////
+  //  <MediasDisplayer {nft} {index} {displayMode}? />
+  // Determin type of media and Display correct component according to the entering parameters of {displayMode}
+  /////////////////////////////////////////////////
+  export let nft: NftType;
+  export let index: number;
+  export let displayMode: { target: string; small: boolean } = { target: "preview", small: true };
```

Comment:
should not use object as props, better pass 2 props

or pass all options as object : not sure it's a good idea for reactivity